lib/IdP/SAML2.php

```diff
@@ -669,11 +674,13 @@ public static function receiveLogoutMessage(IdP $idp)
         $binding = Binding::getCurrentBinding();
         $message = $binding->receive();
 
+        /** @psalm-var null|string|\SAML2\XML\saml\Issuer  Remove in SSP 2.0 */
```

Comment:
Why the psalm-var stuff?

Reply:
That came with the newer version of the base simplesamlphp file.